skunkworks_crow/src/main/java/org/odk/share/bluetooth/BluetoothReceiver.java

```diff
@@ -33,21 +34,28 @@ public void onReceive(Context context, Intent intent) {
             return;
         }
 
-        Timber.d(TAG, action);
-        BluetoothDevice dev = intent.getParcelableExtra(BluetoothDevice.EXTRA_DEVICE);
-        if (dev != null) {
-            Timber.d("%s BluetoothDevice: %s, Address: %s", TAG, dev.getName(), dev.getAddress());
-        }
-        switch (action) {
-            case BluetoothAdapter.ACTION_STATE_CHANGED:
-                int state = intent.getIntExtra(BluetoothAdapter.EXTRA_STATE, 0);
-                Timber.d("%s STATE: %s", TAG, state);
-                break;
-            case BluetoothDevice.ACTION_FOUND:
-                short rssi = intent.getShortExtra(BluetoothDevice.EXTRA_RSSI, Short.MAX_VALUE);
-                Timber.d("EXTRA_RSSI: %s", rssi);
-                bluetoothReceiverListener.onDeviceFound(dev);
-                break;
+        BluetoothDevice bluetoothDevice = intent.getParcelableExtra(BluetoothDevice.EXTRA_DEVICE);
+        if (bluetoothDevice != null) {
+            Timber.d("BluetoothDevice: %s, Address: %s", bluetoothDevice.getName(), bluetoothDevice.getAddress());
+            switch (action) {
+                case BluetoothAdapter.ACTION_STATE_CHANGED:
+                    int state = intent.getIntExtra(BluetoothAdapter.EXTRA_STATE, 0);
+                    Timber.d("STATE: %s", state);
```

Comment:
What is the purpose of this? I guess it is also not being used.

Reply:
removed.